README.md

```diff
@@ -37,38 +37,38 @@ context after the span finished.
 use skywalking::context::tracer::Tracer;
 use skywalking::reporter::grpc::GrpcReporter;
 use std::error::Error;
-use std::sync::Arc;
 use tokio::signal;
 
-async fn handle_request(tracer: Arc<Tracer<GrpcReporter>>) {
+async fn handle_request(tracer: Tracer) {
     let mut ctx = tracer.create_trace_context();
 
     {
-        // Generate an Entry Span when a request
-        // is received. An Entry Span is generated only once per context.
-        let span = ctx.create_entry_span("op1").unwrap();
+        // Generate an Entry Span when a request is received.
+        // An Entry Span is generated only once per context.
```

Comment:
Is this limited through codes? In some rare cases(such as queue bulk consuming), we may face multi-entry spans in one context.

Reply:
Limitation of codes have removed.